Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -1376,8 +1394,8 @@
     <RegEx find="pelova" replaceWith="rica" />
     <RegEx find="([sS])pase(\b|[mšt])" replaceWith="$1pasi$2" />
     <RegEx find="([sS])pasen([aeiou])" replaceWith="$1pašen$2" />
-    <RegEx find="spolja?š?nj?([aeiu])" replaceWith="vanjsk$1" />
-    <RegEx find="Spolja?š?nj?([aeiu])" replaceWith="Vanjsk$1" />
+    <RegEx find="spolja?š?nj?" replaceWith="vanjsk" />
```

Comment:
The question mark applies to the preceding **atom**,

- **`ab(cd|de)?`** : the sub-expression `(cd|de)` is the atom with highest priority (due to the parenthesis). The expression matches `abcd`, `abde`, and `ab`.
- **`ab[cde]?`** : the class `[cde]` is the atom. The expression matches `abc`, `abd`, `abe`, and `ab`.
- **`ab?cd?`** : the letters `b` and `d` are atoms. The expression matches `abcd`, `abc`, `acd`, and `ac`.